compiler/rustc_codegen_llvm/src/intrinsic.rs

```diff
@@ -1599,8 +1625,8 @@ fn generic_simd_intrinsic<'ll, 'tcx>(
             }
         );
 
-        require!(
-            matches!(mask_elem.kind(), ty::Int(_)),
+        let m_elem_bitwidth = require_int_ty!(
+            mask_elem.kind(),
             InvalidMonomorphization::ThirdArgElementType {
```

Comment:
I think this should refer to the first argument, but there is a ui test for it and fixing it probably belongs in a separate PR